projectNEWM/newm-artist-portal
# Cu 2vx4wrm: upload song

## What

- Implements upload song functionality

### Note

- Upload audio file and minting song will be implemented separately

## Look

<img width="1440" alt="Screen Shot 2022-06-04 at 3 02 17 AM" src="https://user-images.githubusercontent.com/5877597/171994615-64da5e4d-3785-44c5-afd2-c79b492c5edd.png">

## Test

1. Upload a song using the form
2. Use the Redux logging to verify the song with the image url is fetched from the NEWM API
